executor/agents/agno/config_utils.py

```
@@ -231,37 +250,35 @@ def extract_agno_options(self, task_data: Dict[str, Any]) -> Dict[str, Any]:
-            else:
-                # Handle single bot object (original logic)
-                logger.info("Found single bot configuration")
+        elif bot_config and isinstance(bot_config, list):
+            # Handle bot array - use the first bot configuration
+            team_members = list(bot_config)
+            options["team_members"] = team_members
+
+            # Log after constructing team_members
+            first_bot_name = "unnamed"
+            if team_members and isinstance(team_members[0], dict):
+                first_bot_name = team_members[0].get("name", "unnamed")
```

Ruff (0.15.2)

[error] Local variable `first_bot_name` is assigned to but never used

Remove assignment to unused variable `first_bot_name`

(F841)